src/nashpy/integer_pivoting/integer_pivoting.py

```diff
@@ -3,9 +3,10 @@
 algorithm.
 """
 import numpy as np
+import numpy.typing as npt
 
 
-def make_tableau(M):
+def make_tableau(M: npt.ArrayLike) -> npt.ArrayLike:
```

Comment:
```suggestion
def make_tableau(M: npt.NDArray) -> npt.NDArray:
```

My mistake! I gave you the wrong original suggestion.